Client/Frontend/Browser/FaviconManager.swift

```diff
@@ -57,43 +57,44 @@ class FaviconManager: TabHelper {
     func getFavicon(_ tab: Tab, iconUrl: URL, currentURL: URL, icon: Favicon, profile: Profile) -> Deferred<Maybe<Favicon>> {
         let deferred = Deferred<Maybe<Favicon>>()
         let manager = SDWebImageManager.shared()
-        let options = tab.isPrivate ?
-            [SDWebImageOptions.lowPriority, SDWebImageOptions.cacheMemoryOnly] : [SDWebImageOptions.lowPriority]
+        let options = tab.isPrivate ? [SDWebImageOptions.lowPriority, SDWebImageOptions.cacheMemoryOnly] : [SDWebImageOptions.lowPriority]
```

Comment:
Nit: these lists: can we use or instead of lists? `.lowPriority | .cacheMemoryOnly` ?